src-newer-jdks/orchard/java/parser_next.clj

```diff
@@ -0,0 +1,299 @@
+(ns orchard.java.parser-next
```

Comment:
You should also mention in the metadata when was this ns added.

Reply:
It wasn't intentional, before I changed the code to something else and then I 'reverted' it.

Anyway, the related conditions in this ns now look more homogeneous now.